spec/rubocop/cop/rspec/empty_line_after_hook_spec.rb

```diff
@@ -19,6 +19,126 @@
     RUBY
   end
 
+  it 'allows multiple one-liner blocks' do
+    expect_offense(<<-RUBY)
+      RSpec.describe User do
+        before { do_something }
+        before { do_something_else }
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Add an empty line after `before`.
+        it { does_something }
+      end
+    RUBY
+
+    expect_correction(<<-RUBY)
+      RSpec.describe User do
+        before { do_something }
+        before { do_something_else }
+
+        it { does_something }
+      end
+    RUBY
+  end
+
+  it 'does not register an offense for chainer one-liner `before` hooks' do
+    expect_no_offenses(<<-RUBY)
+      RSpec.describe User do
+        before { do_something }
+        before { do_something_else }
+
+        it { does_something }
+      end
+    RUBY
+  end
+
+  it 'registers an offense for multiline blocks without empty line after' do
+    expect_offense(<<-RUBY)
+      RSpec.describe User do
+        before do
+          do_something
+        end
+        ^^^ Add an empty line after `before`.
+        before { do_something_else }
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Add an empty line after `before`.
+        it { does_something }
+      end
+    RUBY
+
+    expect_correction(<<-RUBY)
+      RSpec.describe User do
+        before do
+          do_something
+        end
+
+        before { do_something_else }
+
+        it { does_something }
+      end
+    RUBY
+  end
+
+  it 'registers an offense for multiline blocks without empty line before' do
+    expect_offense(<<-RUBY)
+      RSpec.describe User do
+        before { do_something_else }
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Add an empty line after `before`.
+        before do
+          do_something
+        end
+        ^^^ Add an empty line after `before`.
+        it { does_something }
+      end
+    RUBY
+
+    expect_correction(<<-RUBY)
+      RSpec.describe User do
+        before { do_something_else }
+
+        before do
+          do_something
+        end
+
+        it { does_something }
+      end
+    RUBY
+  end
+
+  it 'allows chained one-liner with different hooks' do
+    expect_no_offenses(<<-RUBY)
+      RSpec.describe User do
+        before { do_something_else }
+        after { do_something_else }
+
+        it { does_something }
+      end
+    RUBY
+  end
+
+  context 'when AllowConsecutiveOneLiners option is false' do
+    let(:cop_config) { { 'AllowConsecutiveOneLiners' => false } }
+
+    it 'doesn\'t allow multiple one-liner blocks' do
```

Comment:
Cosmetic: https://rubystyle.guide/#consistent-string-literals-single-quote

Reply:
No blame/shame or else but TBH that's why I don't understand why some projects chose `'` over `"`.
With the later, you don't need to think about it

Reply:
I can only think of those who are too lazy to stretch their pinky to reach the Shift button each time they need a quote. 🤷 😄